.github/workflows/ios_deploy_to_testflight.yml

```diff
@@ -1,50 +1,44 @@
+# TODO Fix this workflow
+
 name: ios-deploy-to-testflight
 on:
-  # Trigger the workflow on push action
   push:
     branches:
       - develop
 
 jobs:
-  build_and_upload_to_testflight:
-    name: Build And Upload iOS Application To TestFlight
-    runs-on: macOS-latest
+  build_and_upload_to_app_store:
+    name: Build and upload iOS app to AppStore
+    runs-on: macOS-12
```

Comment:
Can't we use the latest version of macOS? @ryan-conway 
Let's revert it for this PR. I think to revisit this workflow with the latest version is better. What do you think?